grails-app/controllers/org/broadinstitute/orsp/ConsentGroupController.groovy

```diff
@@ -126,10 +126,18 @@ class ConsentGroupController extends AuthenticatedController {
      * @return Redirect to the project view page
      */
     def breakLink() {
-        def issue = queryService.findByKey(params.projectKey)
-        def links = ConsentCollectionLink.findAllByProjectKeyAndConsentKey(params.projectKey, params.consentKey)
-        deleteCollectionLinks(links)
-        redirect(controller: "project", action: "main", params: [projectKey: params.projectKey, tab: "consent-groups"])
+        try {
+            queryService.updateCollectionLinkStatus(params.consentKey, params.projectKey, params.type == "unlink" ? IssueStatus.Unlinked.name : IssueStatus.Rejected.name)
+            List<ConsentCollectionLink> links = ConsentCollectionLink.findAllByProjectKeyAndConsentKey(params.projectKey, params.consentKey)
+            deleteCollectionLinks(links)
+            response.status = 200
+            render([message: "Links are been deleted"] as JSON)
```

Comment:
nit: It would be a lot nicer to describe for the user what links were deleted. As it stands now, this is a pretty opaque operation.